### PR TITLE
F45I: Rename RGWAdminClient to S3SignedClient

### PR DESCRIPTION
**Pull request by avrnv**
# Problem Statement
- As per discussion with Ujjwal, references to RGW are omitted for the signed S3 client.

# Design
- Rename RGWAdminClient to S3SignedClient

# Coding 
-  Coding conventions are followed and code is consistent [Y/N]: Y
-  Confirm All CODACY errors are resolved [Y/N]: Y

# Testing 
- [x] Confirm that Test Cases are added (for both the cases, fix and feature) [Y/N]: 
- [x] Confirm Test Cases cover Happy Path, Non-Happy Path and Scalability [Y/N]: 
- [ ] Confirm Testing was performed with installed RPM [Y/N]:  

# Review Checklist 
  Before posting the PR please ensure
- [x] PR is self reviewed
- [ ] Is there a change in filename/package/module or signature [Y/N]: 
- [ ] If yes for above point, Is a notification sent to all other cortx components [Y/N]
- [x] Jira is updated
- [x] Check if the description is clear and explained. 
- [ ] Check Acceptance Criterion is defined. 
- [ ] All the tests performed should be mentioned before Resolving a JIRA. 
- [ ] Verification needs to be done before marked as Closed/Verified 

# Documentation
- [ ] Changes done to WIKI / Confluence page

### CODE REVIEW

**Review by pawankumarsrivastava — APPROVED** [review]

**Review by rahul-tripathi-git — APPROVED** [review]
+1

**Review by rahul-tripathi-git — APPROVED** [review]
+1

File: py-utils/src/utils/s3/client.py [hunk 1]
```diff
@@ -21,14 +21,14 @@
 from typing import Any, Dict, Optional, Tuple
 
 from cortx.utils.http import HTTPClient, HTTPClientException
-from cortx.utils.rgwadmin.exceptions import RGWAdminClientException
+from cortx.utils.s3.exceptions import S3SignedClientException
 
 
-class RGWAdminClient(HTTPClient):
+class S3SignedClient(HTTPClient):
```
Comment (ujjwalpl) [inline py-utils/src/utils/s3/client.py:27]: Can we make it simple "S3Client". Signed is kind of implicit. 

Also, please rename HTTPClient to HttpClient as per the canonical naming.

**Review by ujjwalpl — CHANGES_REQUESTED** [review]

**Review by rahul-tripathi-git — CHANGES_REQUESTED** [review]
Kindly fix above comments.

**Review by ujjwalpl — APPROVED** [review]

**Review by rahul-tripathi-git — APPROVED** [review]
+1

**Pull request merged**